src/main/java/frc/utils/GBSubsystem.java

```diff
@@ -29,4 +29,6 @@ public final void periodic() {
 
 	protected abstract void subsystemPeriodic();
 
+	public void setBrake(boolean brake){}
```

Comment:
maybe create MotorSubsystem and use it in it, but your choice if dont think its a must